test/js/api/attachment.js

```diff
@@ -7,7 +7,7 @@ describe("AttachmentAPI", function() {
 
     describe("#getAttachmentOrderUrl()", function() {
         it("should be able to generate a simple attachment URL", async () => {
-            const remote = ripe.RipeAPI();
+            const remote = ripe.RipeAPI({ url: config.TEST_URL });
             const result = remote.getAttachmentOrderUrl(123123123);
             assert.strictEqual(result, "https://sandbox.platforme.com/attachments/123123123/data?");

```

Comment:
Shouldn't this check against `config.TEST_URL` instead of the hardcoded "https://sandbox.platforme.com"?
I think you should have a `config.TEST_WEB_URL` that it's the url without the `/api/` suffix and use that.

For reference:
https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/base/ripe.js

Reply:
Ye this URL is as you said the `webUrl` so it does not make sense to compare it to `config.TEST_URL`.
Because no web url is given as a configuration variable, this always passes. 

Since my use case and this PR concerned only the change of the API URL, I did not allow for web url configuration. I think its best this way, if need be, by necessity, we can allow web url to be changed, maybe even have a `getRipeOptions` method for the tests to avoid repetition. But I wouldn't do it here.